tensorflow/buildspec.yml

```diff
@@ -56,7 +56,7 @@ images:
   BuildTensorflowGpuCu112Py38TrainingDockerImage:
     <<: *TRAINING_REPOSITORY
     build: &TENSORFLOW_GPU_TRAINING_PY3 false
-    image_size_baseline: &IMAGE_SIZE_BASELINE 7738
+    image_size_baseline: &IMAGE_SIZE_BASELINE 7800
```

Comment:
Need to change it for the example image as well. You could make `7880` for both, gpu and example gpu images